### PR TITLE
Upgraded gradle version to 5.6.3

### LINKED ISSUE #231: Upgrade Gradle version
Parseq project uses old version of Gradle and hence we get degraded build performance and debuggability. Please bump Gradle to latest.

### PR DESCRIPTION
**Pull request by hiteshsharma**
Fixes #231

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.8-bin.zip
+distributionUrl=https://services.gradle.org/distributions/gradle-5.6.3-bin.zip
```
Comment (mchen07) [inline gradle/wrapper/gradle-wrapper.properties]: Any special reason we have `https:\` before? @bolobobo

Comment (mchen07) [reply gradle/wrapper/gradle-wrapper.properties]: Are we using some special feature of gradle 5.6.3? @hiteshsharma

Comment (hiteshsharma) [reply gradle/wrapper/gradle-wrapper.properties]: @mchen07 seems `https\://` is broken url. Not sure how it worked. I simply copied the link from https://services.gradle.org/distributions/
No we are not using any 5.6.3 feature for now but someone created issue #231 for upgrading so I just upgraded it and it came out to be straightforward enough as we don't have any backwards incompatible usage

Comment (bolobobo) [reply gradle/wrapper/gradle-wrapper.properties]: @hiteshsharma  Thanks for pointing that out. I just followed the guideline provided by Gradle company. Please look at this doc. https://docs.gradle.org/current/userguide/gradle_wrapper.html I need to double check this with @mockitoguy .

Comment (bolobobo) [reply gradle/wrapper/gradle-wrapper.properties]: @hiteshsharma This file is auto generated by running ` ./gradlew wrapper` and that's probably for URI encoding. We should not edit that file by hand.

**Comment by evanw555** [comment]
We should prioritize getting this fix merged to pick up important fixes (e.g. functionality in the `maven-publish` plugin). I'll test this out soon to see if we can get it through.

**Comment by evanw555** [comment]
Fixed this in PR #252, closing this one.

**Pull request closed without merging**